src/lib.rs

```diff
@@ -1058,6 +1058,27 @@ pub trait Itertools : Iterator {
             None
         }
     }
+
+    /// Sort iterator elements.
+    ///
+    /// **Note:** This consumes the entire iterator, uses the
+    /// **slice::sort_by()** function and returns the sorted vector.
+    ///
+    /// ```
+    /// use itertools::Itertools;
+    ///
+    /// // sort in descending order
+    /// let v = (0..5).into_iter().sort_by(|&a, &b| a.cmp(&b).reverse());
```

Comment:
Thank you. This first into_iter is redundant, maybe the example can demonstrate using sort_by().into_iter() so that users don't miss that possibility?